[2020-05-22 09:25:56,406] - logger.py] - INFO: test_student:点击"网络演训"
[2020-05-22 09:25:56,430] - logger.py] - INFO: test_student:点击"模拟演练"
[2020-05-22 09:25:56,478] - logger.py] - INFO: test_student:修改任务
[2020-05-22 09:25:57,180] - logger.py] - INFO: test_student:等一秒
[2020-05-22 09:25:58,181] - logger.py] - INFO: test_student:点击"场景配置"
[2020-05-22 09:25:58,233] - logger.py] - INFO: test_student:点击"接入配置"
[2020-05-22 09:25:58,778] - logger.py] - INFO: test_student:点击"详情配置"
[2020-05-22 09:25:59,395] - logger.py] - INFO: test_student:获取虚机名称前缀及按钮
[2020-05-22 09:30:39,827] - logger.py] - INFO: test_student:点击"网络演训"
[2020-05-22 09:30:39,851] - logger.py] - INFO: test_student:点击"模拟演练"
[2020-05-22 09:30:39,895] - logger.py] - INFO: test_student:修改任务
[2020-05-22 09:30:40,772] - logger.py] - INFO: test_student:等一秒
[2020-05-22 09:30:41,773] - logger.py] - INFO: test_student:点击"场景配置"
[2020-05-22 09:30:41,810] - logger.py] - INFO: test_student:点击"接入配置"
[2020-05-22 09:30:42,360] - logger.py] - INFO: test_student:点击"详情配置"
[2020-05-22 09:30:42,960] - logger.py] - INFO: test_student:获取虚机名称前缀及按钮
[2020-05-22 09:30:55,311] - logger.py] - ERROR: test_student:【获取虚机名称前缀及按钮】Traceback (most recent call last):
  File "E:\project\Start_in_batches\common\Script.py", line 71, in script_case
    self.driver = script_type[value['execution_type']](loca,value,self.username)#调用操作函数
  File "E:\project\Start_in_batches\common\script_value.py", line 463, in batch_create_vm_log_script
    result = self.find(success_locator,time_out="5").text
  File "E:\project\Start_in_batches\common\base.py", line 20, in find
    element = WebDriverWait(self.driver, int(time_out), 0.5).until(lambda x: x.find_element(*locator))
  File "E:\python\lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


[2020-05-22 09:30:55,311] - logger.py] - INFO: test_student:返回分组列表
[2020-05-22 09:30:55,377] - logger.py] - ERROR: test_student:【返回分组列表】Traceback (most recent call last):
  File "E:\project\Start_in_batches\common\Script.py", line 71, in script_case
    self.driver = script_type[value['execution_type']](loca,value,self.username)#调用操作函数
  File "E:\project\Start_in_batches\common\script_value.py", line 35, in click_script
    self.click(locator)
  File "E:\project\Start_in_batches\common\base.py", line 44, in click
    self.find(locator).click()
  File "E:\python\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "E:\python\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "E:\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "E:\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: Element <span data-v-73a1d177="">...</span> is not clickable at point (1560, 256). Other element would receive the click: <div data-v-73a1d177="" class="shadow" style=""></div>
  (Session info: chrome=81.0.4044.138)
  (Driver info: chromedriver=71.0.3578.80 (2ac50e7249fbd55e6f517a28131605c9fb9fe897),platform=Windows NT 10.0.17763 x86_64)


[2020-05-22 09:30:55,377] - logger.py] - INFO: test_student:保存
[2020-05-22 09:31:00,497] - logger.py] - ERROR: test_student:【保存】Traceback (most recent call last):
  File "E:\project\Start_in_batches\common\Script.py", line 71, in script_case
    self.driver = script_type[value['execution_type']](loca,value,self.username)#调用操作函数
  File "E:\project\Start_in_batches\common\script_value.py", line 35, in click_script
    self.click(locator)
  File "E:\project\Start_in_batches\common\base.py", line 44, in click
    self.find(locator).click()
  File "E:\project\Start_in_batches\common\base.py", line 20, in find
    element = WebDriverWait(self.driver, int(time_out), 0.5).until(lambda x: x.find_element(*locator))
  File "E:\python\lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


[2020-05-22 09:31:00,497] - logger.py] - INFO: test_student:等待6000秒
[2020-05-22 09:46:45,674] - logger.py] - INFO: test_student:点击"网络演训"
[2020-05-22 09:46:45,696] - logger.py] - INFO: test_student:点击"模拟演练"
[2020-05-22 09:46:45,743] - logger.py] - INFO: test_student:修改任务
[2020-05-22 09:46:46,451] - logger.py] - INFO: test_student:等一秒
[2020-05-22 09:46:47,452] - logger.py] - INFO: test_student:点击"场景配置"
[2020-05-22 09:46:47,503] - logger.py] - INFO: test_student:点击"接入配置"
[2020-05-22 09:46:48,059] - logger.py] - INFO: test_student:点击"详情配置"
[2020-05-22 09:46:48,660] - logger.py] - INFO: test_student:获取虚机名称前缀及按钮
[2020-05-22 09:46:49,560] - logger.py] - INFO: test_student:返回分组列表
[2020-05-22 09:46:49,665] - logger.py] - INFO: test_student:保存
[2020-05-22 09:46:49,753] - logger.py] - INFO: test_student:等待6000秒
[2020-05-22 09:56:07,337] - logger.py] - INFO: test_student:点击"网络演训"
[2020-05-22 09:56:07,360] - logger.py] - INFO: test_student:点击"模拟演练"
[2020-05-22 09:56:07,414] - logger.py] - INFO: test_student:修改任务
[2020-05-22 09:56:08,113] - logger.py] - INFO: test_student:等一秒
[2020-05-22 09:56:09,114] - logger.py] - INFO: test_student:点击"场景配置"
[2020-05-22 09:56:09,153] - logger.py] - INFO: test_student:点击"接入配置"
[2020-05-22 09:56:09,709] - logger.py] - INFO: test_student:点击"详情配置"
[2020-05-22 09:56:10,310] - logger.py] - INFO: test_student:获取虚机名称前缀及按钮
[2020-05-22 10:03:22,266] - logger.py] - INFO: test_student:返回分组列表
[2020-05-22 10:03:22,363] - logger.py] - INFO: test_student:保存
[2020-05-22 10:03:22,455] - logger.py] - INFO: test_student:等待6000秒
